client/src/components/MultiJurisdictionAudit/AASetup/Contests/index.tsx

```diff
@@ -69,6 +71,9 @@ const Contests: React.FC<IProps> = ({
   if (!contests) return null // Still loading
   const filteredContests = contests.filter(c => c.isTargeted === isTargeted)
 
+  /* istanbul ignore next */
+  if (isBatch && !isTargeted && nextStage.activate) nextStage.activate() // skip to next stage if on opportunistic contests screen and during a batch audit (until batch audits support multiple contests)
```

Comment:
why is this necessary if you already removed opportunistic contests from the menu items?

Reply:
@jonahkagan Because when you click the Next button on the Targeted Contests stage it goes to the Opportunistic Contests stage even if the menu item isn't displayed. Removing it entirely from the whole menu items code would have been waaaay more work than just skipping by it invisibly.

Reply:
gotcha, makes sense